mods/tuxemon/maps/spyder_bedroom.tmx

```diff
@@ -67,6 +67,7 @@
     <property name="act1" value="translated_dialog spyder_papertown_restinbed"/>
     <property name="act2" value="set_monster_health ,"/>
     <property name="act3" value="set_monster_status ,"/>
+    <property name="act4" value="set_variable teleport_faint:spyder_bedroom.tmx 7 6"/>
```

Comment:
If you never rest, then when you faint you will be teleported to the default target, which is from a different mod!

Please add a new event within spyder_bedroom to default it. I think this will do:

name="act1" value="set_variable teleport_faint:spyder_bedroom.tmx 7 6"
name="cond1" value="not variable_set teleport_faint"

Reply:
You're right